handler.py

```diff
@@ -321,11 +321,11 @@ def main():
 
         try:
             if choice == '1':
-                item_id = int(input("Enter item ID: "))
-                name = input("Enter item name: ")
                 price = float(input("Enter item price: "))
```

Comment:
Add a comment of what it does
```suggestion
                # Prompt the user to enter the item price and convert it to a float
                price = float(input("Enter item price: "))
```